js/map.js

```diff
@@ -112,18 +131,56 @@ function mouseover(d) {
 		.style("top", (y_val-y_buffer)+"px")
 		.text(formatCountyName(d.properties.ID));
   
-  d3.select(this).style('fill', 'orange'); 
+  d3.select(this)
+    .classed('hover', true); 
 }
 
 function mouseout(d) {
   d3.selectAll(".tooltip")
 		.style("display", "none");
 		
-  d3.select(this).style('fill', "#efefef");
+  d3.select(this)
+    .classed('hover', false);
 }
 
 function formatCountyName(nm) {
   return nm.split(",").reverse().join(", ");
 }
 
+function extractPrecipVal(precip_ts, cell_id) {
+  // there's got to be a cleaner way to do this ...
+  var precip_cell = precip_ts.filter(function(i) {
```

Comment:
maybe we could get it so precip_ts is indexed by cell_id (uses cell_id as the key)? not that you need to try it for this PR, just sometime in the future

Reply:
Agree, that would be useful. That's what this line did before in my data load function, `precip.set(d.cell, +d.precip); `. I couldn't figure out how to get it to work after the data was loaded. I think [this page](https://github.com/d3/d3-collection) gets at that, but I just struggled with it and decided to move on for now.